### PR TITLE
docs: add readme with links to community docs #4

### PR DESCRIPTION
**Pull request by johnsBeharry**
#### Problem
- Meta repo purpose may not be clearly communicated.
- Difficult to navigate the bitcoin design resources created so far

#### Solution
Create a README.md

#### Result
Created a README.md with links to process documents and channels of the bitcoin design slack.

### CODE REVIEW

**Comment by GBKS** [comment]
Looks good. Thanks for getting this going.

**Comment by johnsBeharry** [comment]
@GBKS whats the procedure to get this merged? lol

File: README.md [hunk 1]
```diff
@@ -1,2 +1,30 @@
 # Bitcoin Design - Meta
-`community`, `project-management`, `process`
\ No newline at end of file
+`community`, `project-management`, `process`
+
+Anything related to group consensus and coordination of the Bitcoin Design Community.
+
+### Getting Started
+1. Pick existing [project](https://docs.google.com/document/d/1P20rPq6V6w78dXpBqLiiiB-kSypZOzXaG9ZUErYIq28/edit) in the space and do something small
```
Comment (moneyball) [inline README.md:7]: My understanding of our process is that we'll use GitHub Projects, so the link should point to the list of active projects on GitHub not a Google doc

File: README.md [hunk 2]
```diff
@@ -1,2 +1,30 @@
...
+2. If you are passionate. Just do it. Worst case you publish it on your own
+3. Contribute to parts of the design guide and website even if it is a slower process to get things approved
+
```
Comment (moneyball) [inline README.md:10]: IMO we should remove negative language like "even if it is a slower process" ... let's be optimistic when introducing new people to the project :)

File: README.md [hunk 3]
```diff
@@ -1,2 +1,30 @@
...
+### Projects
+
+- Project list — [Doc](https://docs.google.com/document/d/1P20rPq6V6w78dXpBqLiiiB-kSypZOzXaG9ZUErYIq28/edit)
+
```
Comment (moneyball) [inline README.md:14]: same comment as above - need to point to github not google docs

File: README.md [hunk 4]
```diff
@@ -1,2 +1,30 @@
...
+### Governance / Opensource Design
+
+- Slack Channel — [#open-design](https://bitcoindesign.slack.com/archives/C015GFYSJNA)
+- Discussions -- [Issues](https://github.com/BitcoinDesign/Meta/issues)
+
```
Comment (moneyball) [inline README.md:19]: seems like the project lifecycle in the google doc should be placed in markdown on github and then pointed to as part of "governance" (or, process)

File: README.md [hunk 5]
```diff
@@ -1,2 +1,30 @@
...
+1. Pick existing [project](https://docs.google.com/document/d/1P20rPq6V6w78dXpBqLiiiB-kSypZOzXaG9ZUErYIq28/edit) in the space and do something small
+2. If you are passionate. Just do it. Worst case you publish it on your own
+3. Contribute to parts of the design guide and website even if it is a slower process to get things approved
+
+### Projects
+
+- Project list — [Doc](https://docs.google.com/document/d/1P20rPq6V6w78dXpBqLiiiB-kSypZOzXaG9ZUErYIq28/edit)
+
+### Governance / Opensource Design
+
```
Comment (moneyball) [inline README.md:16]: maybe use softer language "process" vs. "governance"?

**Comment by GBKS** [comment]
@johnsBeharry as this is a PR you created, could you please review the outstanding changes @moneyball suggested?

**Pull request merged**